app/controllers/application_controller.rb

```diff
@@ -33,6 +33,12 @@ def current_provider
                           end
   end
 
+  # Returns the default role
+  def default_role
+    default_role_setting = SettingGetter.new(setting_name: 'DefaultRole', provider: current_provider).call
+    @default_role = Role.find_by(name: default_role_setting, provider: current_provider) || Role.find_by(name: 'User', provider: current_provider)
```

Comment:
Sorry for the late review,
What's the use of the `@default_role` ?
Don't we like to raise an exception when we cannot find the User record?
What do you think guys?

Reply:
I'm returning an instance variable for consistency as all other methods in ApplicationControllers are returning an instance variable.

Reply:
Being consistent is good,
However, these variables has no function except they're causing an extra usage of memory that we don't need.